### DISCUSSION TITLE
Tuya_local devices unreachable after stop and restart of container

### DISCUSSION THREAD

**Discussion by rhassold** [make-all/tuya-local#1877] [Q&A]
Hi guys, I like the tuya_local integration a lot and use it for my WIFI thermostats. My HA installtion is running on a Synology DS720+ in a container as part of a project with other containers.
Every night I run a batch job to stop all containers, make a backup of all databases, pull new images and restart the containers.
After that process all tuya_local devices are unreachable. I have to open the tuya_local integration and manually reload the devices, after that the devices are available and work as normally.

Does anybody have an idea as why this happens and what might have gone wrong here?

**Comment by make-all** [comment]
This should be fixed in 2024.5.2
